src/markdown/remark-slate-transformer/transformers/mdast-to-slate.ts

```diff
@@ -364,14 +364,36 @@ function createBreak(node: mdast.Break) {
 
 export type Link = ReturnType<typeof createLink>;
 
+const noteLinkRegex = /^\.\/(?:(.+)\/)?([a-zA-Z0-9-]+)\.md$/;
```

Comment:
Document implications of this. Should this live in the `features/not-ref` directory?